webapp/packages/plugin-sql-editor-new/src/SQLEditor/useSqlDialectAutocompletion.ts

```diff
@@ -29,26 +29,10 @@ export function useSqlDialectAutocompletion(data: ISQLEditorData): [Compartment,
   const optionsRef = useObjectRef({ data });
 
   const [config] = useState<CompletionConfig>(() => {
-    function getOptionsFromProposals(explicit: boolean, word: string, proposals: SQLProposal[]): SqlCompletion[] {
```

Comment:
This will break current logic

https://github.com/user-attachments/assets/c0943038-fab6-4269-9ffe-993e6b6785de

https://github.com/user-attachments/assets/46df1e6d-5da7-41a6-818c-771dfa833a78